docs/general/installation/linux.md

```diff
@@ -5,7 +5,58 @@ description: Install on Linux.
 sidebar_position: 2
 ---
 
-## Alpine Linux
+## Official Methods
```

Comment:
I prefer "Officially Supported Distributions" as the sections are listed in distributions

Reply:
I'm using this phrase because I plan to add a link to docker instructions to this section in a future PR